src/pages/mesh/advanced/secrets.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Secrets management
+description: Learn how to manage secrets for API Mesh for Adobe Developer App Builder.
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Secrets management
+
+API Mesh for Adobe Developer App Builder allows you to manage secrets or variables for a mesh. You can use these secrets in your mesh configuration file to securely manage sensitive information. When creating or updating a mesh, you can provide a separate file that defines your secrets. API Mesh encrypts secrets using AES 512-bit encryption.
```

Comment:
Should variables be mentioned here, since we have .env files for that purpose? Or is that referring to things like $TOKEN and $COMMERCE_API_KEY. I have more questions about this topic below